SquAPI.lua

```diff
@@ -61,8 +56,8 @@ local squapi = {}
 -- these variables can be changed to control certain features of squapi.
 
 
---when true it will automatically tick and update all the functions, when false it won't do that. 
---if false, you can run each objects respective tick/update functions on your own - better control. 
+--when true it will automatically tick and update all the functions, when false it won't do that.<br>
```

Comment:
This `<br>` tag should be replaced with a markdown line break (two spaces)

Reply:
Suggestion because I forgot it
```suggestion
--when true it will automatically tick and update all the functions, when false it won't do that.  
```